website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -759,3 +759,35 @@ The ORCA inventory catalog build script is available for download <a href={useBa
 The ORCA internal reconciliation tables build script is available for download <a href={useBaseUrl('files/build_orca_reconcile.sql')} target="_blank" download>here</a>.
 
 
+## Notes from ORCA schema refinement research
+As part of the research, some refinements were made to the ORCA inventory catalog schema. Some attribute modifications were made only to the `granules` table:
+- `execution_id` has been added for helping with logging and tracing.
+- `archive_location` has been removed since `orca_archive_location` already exists in the `files` table.
+
+#### Refined Granule Metadata Object
+
+| Attribute Name   | Data Type | Description                                                            | Notable Items   |
+| ---------------- | --------- | ---------------------------------------------------------------------- | --------------- |
+| cumulus_granule_id | `string`  | granuleId that is provided by Cumulus.                                 | Must be unique per collectionId |
+| execution_id       | `string`  | Step function execution ID from AWS.                                  | Unique ID automatically generated by AWS |
+| ingest_time      | `timestamp` | Date and time in UTC that the data was originally ingested into ORCA | None |
+| last_update      | `timestamp` | Date and time in UTC that information was updated.                   | Potential time filter for Cumulus reconciliation. |
+
+:::note
+  It is recommended to add tags for `hash` and `hash_type` when the values are available to files that get copied over to S3 bucket.
+:::
+### Refined ERD File & SQL script
+
+The refined ADS ERD for ORCA can be downloaded <a href={useBaseUrl('files/ads_orca_schema_refined.xed')} target="_blank" download>here</a>.
+The refined ORCA inventory catalog build script is available for download <a href={useBaseUrl('files/build_orca_inventory_refined.sql')} target="_blank" download>here</a>.
+
+### Addressing open questions
+
+- What is the proper date to use when receiving a date range from Cumulus for a reconciliation report?
```

Comment:
Cleanup comments and items here with the decision that was made to not include acquisition date initially and why.

Reply:
modified the comment